app/assets/stylesheets/settings.scss

```diff
@@ -367,31 +375,31 @@
     }
     .github-repo-row-name {
       font-weight:bold;
+      button {
+        cursor:pointer;
+        font-size:1.2em;
+        font-weight: bold;
+        width: 130px;
+        background: white;
+        border:0px solid;
+        border-radius:3px;
+        padding: 3px 0px;
+        margin-top:-5px;
+        float: right;
+        &:hover, &:focus {
```

Comment:
I prefer this over the previous version since it makes selecting/removing a lot more obvious than the opacity change on a white background.